src/ai/backend/manager/models/keypair.py

```diff
@@ -79,7 +79,13 @@
         onupdate=sa.func.current_timestamp(),
     ),
     sa.Column("last_used", sa.DateTime(timezone=True), nullable=True),
-    sa.Column("rate_limit", sa.Integer),
+    sa.Column(
+        "rate_limit",
+        sa.Integer,
+        nullable=False,
+        default=DEFAULT_KEYPAIR_RATE_LIMIT,
+        server_default=str(DEFAULT_KEYPAIR_RATE_LIMIT),
```

Comment:
Shouldn't we remove `str()` here as the database field is an integer and the original constant is also an int?

Reply:
`server_default=DEFAULT_KEYPAIR_RATE_LIMIT,` will raise error when we run migration. And as I tested, the default value is injected as we want.
The error log is like below.
```
sqlalchemy.exc.ArgumentError: Argument 'arg' is expected to be one of type '<class 'str'>' or '<class 'sqlalchemy.sql.elements.ClauseElement'>' or '<class 'sqlalchemy.sql.elements.TextClause'>', got '<class 'int'>'
```

Reply:
Ah, so it expects a string to be interpolated into the actual SQL statement.
I think it would be better to use `sa.text()` or similar stuffs to reveal that this is an SQL-injected text.

Reply:
`sa.text(DEFAULT_KEYPAIR_RATE_LIMIT)` raises error like below
```
TypeError: expected string or bytes-like object, got 'int'
```
so I just wrap it like this `sa.text(str(DEFAULT_KEYPAIR_RATE_LIMIT))`